modules/platforms/dotnet/Apache.Ignite.Tests/Proto/BinaryTuple/BinaryTupleTests.cs

```diff
@@ -645,6 +645,120 @@ public void TestDefault()
             Assert.AreEqual(default(LocalDateTime), reader.GetDateTime(0));
         }
 
+        [Test]
+        public void TestGetNullable()
+        {
+            var reader = BuildAndRead((ref BinaryTupleBuilder b) => b.AppendNull());
+
+            Assert.IsNull(reader.GetIntNullable(0));
+            Assert.IsNull(reader.GetByteNullable(0));
+            Assert.IsNull(reader.GetShortNullable(0));
+            Assert.IsNull(reader.GetLongNullable(0));
+            Assert.IsNull(reader.GetDoubleNullable(0));
+            Assert.IsNull(reader.GetFloatNullable(0));
+            Assert.IsNull(reader.GetDecimalNullable(0, 123));
+            Assert.IsNull(reader.GetNumberNullable(0));
+            Assert.IsNull(reader.GetStringNullable(0));
+            Assert.IsNull(reader.GetBitmaskNullable(0));
+            Assert.IsNull(reader.GetGuidNullable(0));
+            Assert.IsNull(reader.GetBytesNullable(0));
+            Assert.IsNull(reader.GetDurationNullable(0));
+            Assert.IsNull(reader.GetPeriodNullable(0));
+            Assert.IsNull(reader.GetTimeNullable(0));
+            Assert.IsNull(reader.GetDateNullable(0));
+            Assert.IsNull(reader.GetDateTimeNullable(0));
+        }
+
+        [Test]
+        public void TestAppendNullable()
+        {
+            var guid = Guid.NewGuid();
```

Comment:
Minor

Personally I like having all the values defined in advance, but here you are mixing some if them with raw literals. For Guid could be  Guid.Empty, for DateTime -> MaxVal, etc.

Reply:
I think this looks clear enough as it is. Extracting `1` into a constant just makes the test longer.

Some things have to be defined upfront - you can't inline `Guid.NewGuid` or `UtcNow`, so we do that when necessary.